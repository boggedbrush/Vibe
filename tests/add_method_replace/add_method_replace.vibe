# VibeSpec: 1.2
patch_type: add_method
file: hello.py
class: Greeter
--- code: |
    def greet(self):
        print("Greeter 2.0 at your service!")
