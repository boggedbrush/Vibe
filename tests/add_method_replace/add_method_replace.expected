# hello.py
class Greeter:
    def __init__(self, name):
        print(f"Hello, {name}!")

    def greet(self):
        print("Greeter 2.0 at your service!")

